UIの変更

s-natsuyama opened the pull request:
Segmented ControlをTabBarControllerに変更しました。
NavigationBarのタイトルなどの文字色も変更しています。

Commit:
NavigationVarからTabBarControllerに変更

Commit:
NavigationItemをTabBarControllerに変更

taiju59 commented:
@s-natsuyama 

以下の点について確認お願いします。
だいぶ複雑になってきた感じがあるので、
作り直して、storyboard を分けましょうか。
すぐ解決できそうな見込みがあれば別です。
- Favorites の Navigation Bar の shadow が消えている
- ストリートビュー画面のボタンが黒い (本来は白)
- Favorites から設定画面へ遷移して戻ると Pogs 画面になる
- 長押しで何も起こらない(共有済み)

s-natsuyama commented:
・Favorites の Navigation Bar の shadow が消えている
・ストリートビュー画面のボタンが黒い (本来は白)
・Favorites から設定画面へ遷移して戻ると Pogs 画面になる

上記の３つを今日中に修正し、再度プルリクします。

・長押しで何も起こらない(共有済み)

この問題に関しては青木さんにお願いします。宜しくお願いします。

taiju59 commented:
@s-natsuyama 
すみません、あと追加で2つお願いします。
- Favorite -> ストリートビューへの遷移の peek&pop を有効に
- Favorite -> 設定への遷移の peek&pop を無効に

s-natsuyama commented:
了解しました。

taiju59 commented:
@s-natsuyama 

> 長押しで何も起こらない(共有済み)

詳細理解は微妙ですが、これの対応方法がわかりました。
master に動くものをマージするため、このブランチでの対応をお願いします。
- tableView -> Long Press Gesture Recognizer で  
  gestureRecognizer を紐付け (Ctrl + ドラッグのやつ)
- Long Press Gesture Recognizer -> tableView で  
  delegate を紐付け  (これも Ctrl + ドラッグのやつ)

s-natsuyama commented:
なんとか修正できました。遅くなってすみません。今からプルリクします！

taiju59 commented:
@s-natsuyama 
プッシュすればそれで OK です。
長押し問題も一緒に対応お願いします。(コミットは別で OK)

s-natsuyama commented:
プッシュだけしたのですが、反映されてますか？

taiju59 commented:
@s-natsuyama 
あれ？反映されてませんね。
SourceTree上でも進んでないので、コミットしただけとかじゃないですか？

s-natsuyama commented:
もう一度試してみます！

Commit:
プルリク修正

・Favorites の Navigation Bar の shadow が消えている
・ストリートビュー画面のボタンが黒い (本来は白)
・Favorites から設定画面へ遷移して戻ると Pogs 画面になる
・長押しで何も起こらない(共有済み)

s-natsuyama commented:
お願いします！

taiju59 commented on Pogger/PointCell.swift:
```diff
@@ -130,7 +130,7 @@ class PointCell: UITableViewCell {
         }
         self.shareButton.setTitle(Prefix.iconShare, for: UIControlState())
-        self.optionButton.setTitle(Prefix.iconOption, for: UIControlState())
+        self.optionButton.setTitle("\u{e603}", for: UIControlState())
```
iconFont の指定はベタ書きではなく Prefix に記載からの参照でお願いします

taiju59 commented on Pogger/AppDelegate.swift:
```diff
@@ -19,10 +19,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
-        UINavigationBar.appearance().titleTextAttributes = [NSForegroundColorAttributeName:UIColor.white]
-        UINavigationBar.appearance().tintColor = UIColor.white
+        UINavigationBar.appearance().titleTextAttributes = [NSForegroundColorAttributeName:UIColor.black]
+        UINavigationBar.appearance().tintColor = UIColor.black
```
デフォルトの色指定になったと思うので、この記述は削除で OK です

taiju59 commented on Pogger/ListViewContrtoller.swift:
```diff
@@ -222,27 +219,28 @@ class ListViewController: ViewController, UITableViewDataSource, UITableViewDele
+                actionSheet.addAction(mapOpenAction)
+                actionSheet.addAction(cancelAction)
+                self.present(actionSheet, animated: true, completion: nil)
+            }
```
ここの辺りインデントがずれているので元に戻すようお願いします。
(Favorite もコピペなら同じかも？)

taiju59 commented on Pogger/ListViewContrtoller.swift:
```diff
@@ -289,18 +287,13 @@ class ListViewController: ViewController, UITableViewDataSource, UITableViewDele
+    /*@IBAction func returnListViewForSegue(_ segue: UIStoryboardSegue) {
 
-    }
+    }*/
```
不要となった関数はコメントアウトではなく削除でお願いします

taiju59 commented on Pogger/StreetViewController.swift:
```diff
@@ -38,4 +39,8 @@ class StreetViewController: UIViewController, GMSPanoramaViewDelegate {
                 self.headerView.alpha = self.headerView.alpha == 0 ? 1:0
         })
     }
+    @IBAction func closeButton(_ sender: AnyObject) {
```
メソッド名は"動作"を表したいので変更をお願いします。
didTapCloseButton など

taiju59 commented on Pogger/PointCell.swift:
```diff
@@ -31,10 +31,10 @@ class PointCell: UITableViewCell {
     @IBOutlet weak var favoriteButton: UIButton!
     @IBOutlet weak var shareButton: UIButton!
     @IBOutlet weak var optionButton: UIButton!
+    @IBOutlet weak var dateLabel: UILabel!
```
細かい指摘で恐縮なんですが、意図していない変更は混乱の元なので
もし見つけたら元に戻しておいてください。
(dateLabel を最後に持ってきたことに意図があれば別です！)

taiju59 commented on Pogger/SettingViewController.swift:
```diff
@@ -37,8 +38,7 @@ class SettingViewController: UITableViewController {
 
-    @IBAction func returnSettingViewForSegue(_ segue: UIStoryboardSegue) {
-
+    @IBAction func closeSettingButton(_ sender: AnyObject) {
```
これも動作を表す感じな名前に変更お願いします。
↓と統一感のある名前にしてもらえたら嬉しいです。
https://github.com/taiju59/Pogger/pull/30/files#r81534940

s-natsuyama commented:
一通り修正しました。ビルド確認次第、再度プルリクします。

Commit:
プルリクの再度修正
